todo/src/storage.js

```diff
@@ -0,0 +1,17 @@
+export const loadState = () => {
+  try {
+    const serializedState = localStorage.getItem('state');
+    return serializedState === null ? undefined : JSON.parse(serializedState);
+  } catch (err) {
+    return undefined;
```

Comment:
great implementation of try/catch blocks to handle saving your storage for later invocation